Basic/DyanmicPrefabs/Assets/Scripts/Shared/AppController.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Game
+{
+    public sealed class AppController : NetworkBehaviour
```

Comment:
comments on the purpose of this class please
(I'll stop posting this, if you could look at each class and make sure they have some doc associated, that'd be great)